motion-1/Overview.bs

```diff
@@ -97,7 +97,7 @@ ISSUE: Add more details and examples.
 
 <pre class='propdef'>
 Name: offset-path
-Value: <<url>> | <<angle>> <<size>>? && contain? | [ <<basic-shape>> | <<path()>> ] || <<geometry-box>> | none
+Value: <<url>> | ray(<<angle>> && <<size>>? && contain?) | <<basic-shape>> && <<geometry-box>>? | <<path()>> | none
```

Comment:
How about modifying this part as
`[ < basic-shape > || < geometry-box > ]` ?

< geometry-box > can be specified by itself, referring to 'shape-outside'[1]  or 'clip-path'[2]. But "<< geometry-box >>?" seems that << geometry-box >> is an optional value.

[1] https://www.w3.org/TR/css-shapes-1/#propdef-shape-outside
[2] https://drafts.fxtf.org/css-masking-1/#propdef-clip-path